### PR TITLE
implement todos-1

### PR DESCRIPTION
**Pull request by dragos-rebegea**
- removed ConvertFromByteSliceToArray from convertors and used AddressHandler method instead

### CODE REVIEW

File: core/converter/converters.go [hunk 1]
```diff
@@ -44,16 +44,6 @@ func (ac *addressConverter) IsInterfaceNil() bool {
 	return ac == nil
 }
 
-// TODO - move this as a method in AddressHandler
```
Comment (iulianpascalau) [inline core/converter/converters.go:47]: TODO should have described more. - My bad

File: core/converter/converters.go [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package core
+package converter
```
Comment (iulianpascalau) [inline core/converter/converters.go:1]: the package can be named `converters`

File: testsCommon/bridge/constants.go [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
+import (
+	"errors"
+
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/data"
+)
 
 var notImplemented = errors.New("method not implemented")
+
+// StaticAddress is an instance of Address Handler
+var StaticAddress = data.NewAddressFromBytes([]byte{})
```
Comment (iulianpascalau) [inline testsCommon/bridge/constants.go:12]: we do not need this

**Review by iulianpascalau — COMMENTED** [review]
we can re-target this PR on the feat/bridge-v2 directly

**Review by sstanculeanu — APPROVED** [review]

File: clients/ethereum/client_test.go [hunk 4]
```diff
@@ -220,14 +221,14 @@ func TestClient_GetBatch(t *testing.T) {
 							TokenAddress: token1,
 							Amount:       big.NewInt(20),
 							Depositor:    from1,
-							Recipient:    core.ConvertFromByteSliceToArray(recipient1),
+							Recipient:    data.NewAddressFromBytes(recipient1).AddressSlice(),
```
Comment (iulianpascalau) [inline clients/ethereum/client_test.go:224]: `recipient1.AddressSlice()` and make 
`recipient1 := testsCommon.CreateRandomElrondAddress()`

valid also for recipient 2

File: integrationTests/relayers/ethToElrond_test.go [hunk 5]
```diff
@@ -37,11 +38,11 @@ func TestRelayersShouldExecuteTransferFromEthToElrond(t *testing.T) {
 	ticker2 := "tck-000002"
 
 	value1 := big.NewInt(111111111)
-	destination1 := testsCommon.CreateRandomElrondAddress()
+	destination1 := testsCommon.CreateRandomElrondAddress().AddressBytes()
```
Comment (iulianpascalau) [inline integrationTests/relayers/ethToElrond_test.go:41]: why this change? If you would have kept the original form, the  L65 & L72 could have been:
```
destination1.AddressSlice()
```

Comment (dragos-rebegea) [reply integrationTests/relayers/ethToElrond_test.go]: changed back

File: clients/ethereum/client_test.go (hunk 4 above)
Comment (dragos-rebegea) [reply clients/ethereum/client_test.go]: done

**Review by iulianpascalau — APPROVED** [review]

**Review by sstanculeanu — APPROVED** [review]
